tests/models/whisper/test_modeling_whisper.py

```diff
@@ -2866,6 +2833,7 @@ def test_whisper_longform_single_batch_beam(self):
             "compression_ratio_threshold": 1.35,
             "condition_on_prev_tokens": True,
             "logprob_threshold": -1.0,
+            "renormalize_logits": True,  # necessary to match OAI beam search implementation
```

Comment:
Great! Let's not forget to mention this somewhere in the docs

Reply:
I am thinking about setting it by default to True in Whisper's generate (and adding it to the doc this way) and remove it from here. WDYT?

Reply:
I think it would make more sense to leave it like that here, and change it (setting it by default to True in Whisper's generate-and adding it to the doc this way) in [this PR](https://github.com/huggingface/transformers/pull/34135), since those changes are intended for it anyway.